### PR TITLE
Add .col and .row-* classes to support flex-direction: column. Issue #20233

### PR DESCRIPTION
**Pull request by Mathdrquinn**
PR initially started by addressing [issue #20233](https://github.com/twbs/bootstrap/issues/20233).  I added additional ording and positioning classes to flexbox utilities.

### CODE REVIEW

File: scss/utilities/_flex.scss [hunk 1]
```diff
@@ -4,25 +4,51 @@
...
+        .flex-#{$breakpoint}-wrap { flex-wrap: wrap; }
+        .flex-#{$breakpoint}-no-wrap { flex-wrap: nowrap; }
+      }
+    }
+
     // Flex column reordering
     @include media-breakpoint-up($breakpoint) {
       .flex-#{$breakpoint}-first { order: -1; }
-      .flex-#{$breakpoint}-last { order: 1; }
+      .flex-#{$breakpoint}-last { order: $grid-columns + 1; }
```
Comment (cvrebert) [inline scss/utilities/_flex.scss]: Seems questionable since these utilities aren't specifically grid-related.

**Comment by cvrebert** [comment]
Since this doesn't currently add row or column classes, it seems unrelated to #20233.

**Comment by Mathdrquinn** [comment]
I can pull them off into another or. Sound preferable?  Which ones  do you think should stay/go?

**Comment by Mathdrquinn** [comment]
I can pull them off into another pr. Sound preferable? Which ones do you think should stay/go?

**Comment by cvrebert** [comment]
Multiple PRs would be preferable.

**Comment by Mathdrquinn** [comment]
@cvrebert is this now preferable?

**Comment by Krrish123** [comment]
thanks...@#

**Comment by mdo** [comment]
Totally missed this with recent flex updates! I tackled this in #21407. Thanks though!

**Pull request closed without merging**